### PR TITLE
step of activating the virtual environment added

### LINKED ISSUE #1175: Missing virtual environment activation step in CONTRIBUTING.md
### Description

Contributing markdown file does not have step to activate the virtual environment. If anyone follows the guide step by step, they miss out the activation step.

### What should have happened?

All steps should be there in CONTRIBUTING.md for smooth and proper on-boarding.

### What browser(s) are you seeing the problem on?

_No response_

### Further details

_No response_

### PR DESCRIPTION
**Pull request by shiv-aay**
Closes #1175
  
## Description
Added virtual environment activation step in CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -42,9 +42,11 @@ To install all required modules, complete the following steps:
 1. make sure you are in the repository root directory
 2. create a Python 3.7+ virtual environment (named `env`)
     * `python -m venv env`
-3. use the following command in project dir:
+3. activate the virtual environment using:
```
Comment (gorkemarslan) [inline CONTRIBUTING.md]: The way of activating virtual environments is different for each operating system (Mac Os - Linux and Windows). You only mentioned the step belonging to Windows. Could you add the steps for others? This [link](https://python.land/virtual-environments/virtualenv) might be useful.

You can use the template below. Of course, customization is necessary for some parts like `For Windows users`. You can customize it to be heading.

```
3. activate the virtual environment using:
For windows users:

For MacOS and Linux users:
```

It's also fine to look at other projects to learn how they write activating venv part.

**Review by gorkemarslan — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (shiv-aay) [reply CONTRIBUTING.md]: Yeah, i'll do it.

Comment (shiv-aay) [reply CONTRIBUTING.md]: @gorkemarslan I am thinking of putting all the commands of [Install Requirements](https://github.com/CiviWiki/OpenCiviWiki/blob/develop/CONTRIBUTING.md#install-requirements) section in fenced code blocks by placing triple backticks ```, just like other sections of the file.

Comment (gorkemarslan) [reply CONTRIBUTING.md]: It is a bit hard to imagine it. Could you commit what you think or write down here? 😄

Comment (shiv-aay) [reply CONTRIBUTING.md]: @gorkemarslan I've updated my PR with the mentioned changes. Kindly review.

**Review by gorkemarslan — APPROVED** [review]

**Comment by gorkemarslan** [comment]
@all-contributors please add @shiv-aay for docs

**Pull request merged**